contracts/precompiledContracts.go

```diff
@@ -0,0 +1,224 @@
+package contracts
+
+import (
+	"crypto/ecdsa"
+	"encoding/binary"
+	"encoding/hex"
+	"fmt"
+	"math/big"
+	"math/rand"
+
+	"github.com/ethereum/go-ethereum/accounts/abi/bind"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/common/math"
+	ethtypes "github.com/ethereum/go-ethereum/core/types"
+	"github.com/ethereum/go-ethereum/crypto"
+
+	"github.com/rs/zerolog/log"
+)
+
+func GenerateECRecoverInput(privateKey *ecdsa.PrivateKey) []byte {
+	message := []byte("Test ecRecover")
+	messageHash := crypto.Keccak256Hash(message)
```

Comment:
One thing I would like to be able to test here are empty values. In the future maybe we can have a table with some test values. see https://github.com/0xPolygon/polygon-edge/pull/1260/files#diff-f9f7ba4c9a0e5e981794760f2c0bebe923604444b918224398d14a16d89b2172R150

Reply:
+1, I really like the idea of using different load testing inputs. Tracking for future effort here: [DVT-642](https://polygon.atlassian.net/browse/DVT-642)